docs/enhancements/api-for-skipping-validations.md

```diff
@@ -0,0 +1,220 @@
+
+Disadvantages:
+    This would not help if there's just a bug in a validation, especially if that bug causes the whole install process to stall or fail.
+    Requires a level of manual intervention that may not be desirable in some scenarios. (Could be mitigated with an "ExpectedResult" for the failed validations, if the validation fails in an expected way, proceed with install...)
+
+#### Option 2: Actually skip the validations that we have chosen to skip and provided that everything else passes, allow the installation to proceed.
+
+    All validations run with the exception of the ones marked as `skipped`
+    If any validations fail then the customer must fix the root issue or add a skip for the additionally failing validation.
+
+Advantages:
+    Skipped validations do not execute, therefore the validation process should be quicker.
+    Allows the skipping of bug/panic in a particular validator.
+    Would not require manual intervention.
+
+Disadvantages:
+    Increased risk that the user will miss some important failure due to a skipped validation.
+
+### Effect on validation.
+
+When a validation is skipped then this should be reflected in the Validation status. For example;
+```
+{
+    "id": "api-vips-defined",
+    "status": "skipped",
+    "message": "API virtual IPs validation has been suppressed by the user"
+}
+```
+
+### Effect on Triage tickets.
+To ensure that there is less noise for Triage tickets, any cluster that has events consistent with this sort of override should be omitted from Triage ticketing, in a similar way to how this is done for jupiter installations.
+
+Example here: https://github.com/openshift-assisted/assisted-installer-deployment/blob/463dc6612c923c46d47e80762c72651758a690d3/tools/create_triage_tickets.py#L64
+
+### Per customer feature enablement.
+This feature is to be enabled on a per customer basis, so will need to use the "per customer feature enablement" feature.
+
+See the following link for more information.
+
+https://github.com/openshift/assisted-service/blob/master/docs/dev/feature-per-customer.md
+
+### User Stories
+
+#### Story 1
+
+As a user with known validation issues that need to be bypassed, I would like to be able to bypass VIP validation in cases where VIPS could be incorrectly shown as busy. So that I may proceed with installation.
+
+### Risks and Mitigations
+
+- We should not add uneccesary noise to triage tickets with this, it is therefore quite important to add some signatures to indicate that these validations have been bypassed or to handle in a way that ensures they never appear as Triage tickets.
+
+### Open Questions
+
+N/A
+
+### UI Impact
```

Comment:
If we go with "Option 1: Run all validations, allow the user to review but proceed anyway." then there will be no "skipped" value and no UI impact, right?

Reply:
That's very true